bucket/qv2ray.json

```
@@ -22,15 +22,18 @@
         "if (-not (Test-Path \"$persist_dir\\config\\Qv2ray.conf\")) {",
         "    New-Item \"$dir\\config\" -ItemType Directory | Out-Null",
         "    New-Item \"$dir\\config\\QV2RAY_FEATURE_DISABLE_AUTO_UPDATE\" | Out-Null",
-        "    $current = (appdir 'v2ray' $global) | Join-Path -ChildPath 'current'",
-        "    @{",
-        "        'config_version' = 14",
-        "        'logLevel' = 2",
-        "        'kernelConfig' = @{",
-        "            'v2CorePath_win'   = \"$current\\v2ray.exe\"",
-        "            'v2AssetsPath_win' = \"$current\"",
-        "        }",
-        "    } | ConvertTo-Json -Depth 20 | Out-File \"$dir\\config\\Qv2ray.conf\" -Encoding ascii",
+        "    $v2ray_dir = appdir 'v2ray' $global",
+        "    if (Test-Path $v2ray_dir) {",
+        "        $current = $v2ray_dir | Join-Path -ChildPath 'current'",
+        "        @{",
+        "            'config_version' = 14",
+        "            'logLevel' = 2",
+        "            'kernelConfig' = @{",
+        "                'v2CorePath_win'   = \"$current\\v2ray.exe\"",
+        "                'v2AssetsPath_win' = \"$current\"",
+        "            }",
+        "        } | ConvertTo-Json -Depth 20 | Out-File \"$dir\\config\\Qv2ray.conf\"",
+        "    }",
         "}"
```

Completely ignoring default configuration in case of v2ray is not installed via scoop is not really good idea.

```suggestion
        "if (!(Test-Path \"$persist_dir\\config\\Qv2ray.conf\")) {",
        "    ensure \"$dir\\config\" | Out-Null",
        "    New-Item \"$dir\\config\\QV2RAY_FEATURE_DISABLE_AUTO_UPDATE\" | Out-Null",
        "    $config = @{",
        "        'config_version' = 14",
        "        'logLevel' = 2",
        "    }",
        "    $v2ray_dir = appdir 'v2ray' $global | Join-Path -ChildPath 'current'",
        "    if (Test-Path $v2ray_dir) {",
        "         $config.Add('kernelConfig', @{",
        "             'v2CorePath_win'   = \"$v2ray_dir\\v2ray.exe\"",
        "             'v2AssetsPath_win' = \"$v2ray_dir\"",
        "         })",
        "    }",
        "    ConvertTo-Json $config -Depth 20 | Out-File \"$dir\\config\\Qv2ray.conf\" -Encoding Ascii",
        "}"
```

---

Qv2ray will generate a default config file if one is not found. This is the default behavior and won't cause issues.